operator/api/server.go

```diff
@@ -140,8 +141,14 @@ func (s *Server) StartServer() error {
 		go func() {
 			err := srv.Serve(ln)
 			if err != nil {
-				errCh <- err
-				errs <- err
+				// If the error is due to the server being shutdown, then send nil to
+				// the server errors channel.
+				if errors.Is(err, http.ErrServerClosed) {
+					errs <- nil
```

Comment:
might be worth printing out a message stating "server closed" or something like that.